tensorrt_llm/_torch/pyexecutor/py_executor_creator.py

```
@@ -339,10 +339,10 @@ def create_py_executor(
             _ExecutorCreationStage.GUIDED_DECODER):
         guided_decoder: Optional[GuidedDecoder] = None
         if executor_config.guided_decoding_config is not None:
-            if spec_config is not None and not has_spec_drafter:
-                raise ValueError(
-                    "Guided decoding is only supported with speculative decoding that has a dedicated drafter (two-model engine)."
-                )
+            # if spec_config is not None and not has_spec_drafter:
+            #     raise ValueError(
+            #         "Guided decoding is only supported with speculative decoding that has a dedicated drafter (two-model engine)."
```

Ruff (0.12.2)

Line too long (132 > 120)

(E501)